root/static/scripts/common/constants/locales.json

```diff
@@ -595,7 +623,6 @@
   "ro_RO":"Romanian Romania",
   "rof":"Rombo",
   "rof_TZ":"Rombo Tanzania",
-  "root":"Root",
```

Comment:
We somehow had one use of this, fixing.

Reply:
It doesn’t seem that you have fixed anything, but as mentioned in the PR’s description, I planned to perform this task AFTER the deployment to beta.

Reply:
Hmm. I changed the "root" use I found for artist to English, but I didn't notice your comment above, sorry :)

Reply:
Hmm. My artist was a different one though, so I guess it was added in the meantime!

Reply:
Since you publish no link to any change you made, it is difficult to double-check.